lang/python/python-dpkt/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2017 OpenWrt.org
```

Comment:
Assign copyright to yourself, or omit it unless you're affiliated with OpenWrt..org